packages/lab/src/query-input/internal/CategoryListItem.tsx

```diff
@@ -33,19 +24,15 @@ export const CategoryListItem: FC<CategoryListItemProps> =
       [context.width]
     );
 
-    const onMouseMove: MouseEventHandler = useCallback(() => {
-      props.onMouseMove(category, index);
-    }, [props.onMouseMove, category, index]);
-
     return (
-      <ListItem item={category} onMouseMove={onMouseMove}>
+      <ListItem {...props} label={category?.name}>
```

Comment:
It's always going to be present in this instance. Problem trying to type it more precisely here is that ListItemType is used to type all ListItems. Declarative list items may have no associated item, so its optional. 
I feel like it would complicate the typing quite a lot to accommodate this ?